numba/targets/arrayobj.py

```diff
@@ -1706,6 +1706,60 @@ def scalar_sinc_impl(val):
                                    locals=dict(c=scalar_dtype))
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
+@builtin
+@implement(numpy.angle, types.Kind(types.Number))
```

Comment:
I'll do the move when I've patched up everything else in the review so that the history is cleaner.